study_guides/full_stack_project/full_stack_project_wiki_example/sample_state.md

```diff
@@ -0,0 +1,105 @@
+```js
```

Comment:
For this sample state shape in general, it might be helpful to have some more realistic sample strings. Might help students conceptualize things better

Reply:
agreed, maybe include an example for an array of ids due to an association